app/src/main/java/de/ur/explure/views/ProfileFragment.kt

```diff
@@ -1,6 +1,153 @@
 package de.ur.explure.views
 
+import android.app.AlertDialog
+import android.content.Context
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.TypedValue
+import android.view.View
+import android.widget.Toast
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.constraintlayout.widget.ConstraintSet
 import androidx.fragment.app.Fragment
+import androidx.navigation.Navigation
+import com.google.android.material.textfield.TextInputEditText
+import com.google.android.material.textfield.TextInputLayout
 import de.ur.explure.R
+import de.ur.explure.viewmodel.ProfileFragmentViewModel
+import kotlinx.android.synthetic.main.fragment_profile.*
+import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class ProfileFragment : Fragment(R.layout.fragment_profile)
+const val HORIZONTAL_MARGIN_EDIT_TEXT = 32
```

Comment:
Solche Konstanten sollten eig niemals im Code stehen. Wenn du sie aber trotzdem brauchst dann leg innerhalb der Klasse in `companion object`an:

```
class ProfileFragment : Fragment(R.layout.fragment_profile) {

   companion object {
       private const val VERTICAL_MARGIN_EDIT_TEXT = 8
       private const val HORIZONTAL_MARGIN_EDIT_TEXT = 32
  }
....
}